src/templates/next/components/TableOfContents/TableOfContents.tsx

```diff
@@ -0,0 +1,22 @@
+import { BiRightArrowAlt } from "react-icons/bi"
+import { TableOfContentsProps } from "~/common"
+
+const TableOfContents = ({ items }: TableOfContentsProps) => {
+  return (
+    <div className="flex flex-col gap-3 pl-5 border-l-4 border-content w-[18.75rem]">
+      <p className="text-lg text-content-strong font-semibold">On this page</p>
```

Comment:
not too fussed on this but do check with design if this should be customisable

Reply:
yup it's not customisable